examples/loader-compat/package.json

```diff
@@ -10,6 +10,9 @@
   "keywords": [],
   "author": "",
   "license": "ISC",
+  "dependencies": {
+    "iconv": "3.0.1"
+  },
```

Comment:
I would pick another native module since `iconv` tries to load a binary that is not shipped as described inhttps://github.com/web-infra-dev/rspack/issues/3039#issuecomment-1537132466